tests_metricflow/snapshots/test_metric_time_without_metrics.py/SqlQueryPlan/DuckDB/test_dimensions_with_time_constraint__plan0.sql

```diff
@@ -204,6 +204,7 @@ FROM (
           , subq_1.ds__extract_day
           , subq_1.ds__extract_dow
           , subq_1.ds__extract_doy
+          , subq_1.ds__martian_day
```

Comment:
Ah, this is why we use the same name, so we're setting the standard that the base grain is encoded in the name in all cases.